src/library/impl/meta_reader/schema.h

```diff
@@ -113,6 +113,7 @@ namespace xlang::meta::reader
         auto ParamList() const;
         auto CustomAttribute() const;
         auto Parent() const;
+		auto GenericParam() const;
```

Comment:
spaces not tabs

Reply:
Sory, this is the default in VC++ editor ...

Reply:
I know, I love having to change the default every time I install VS :(

Reply:
done :)

Reply:
a `.editorconfig` in the root of this project to avoid this problem. here is how [VS supports this](https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2017#supported-settings) VS's support for this